app/scripts/lib/setupSentry.js

```diff
@@ -502,60 +501,87 @@ export default function setupSentry({ release, getState }) {
       new FilterEvents({ getMetaMetricsEnabled }),
       new Dedupe(),
       new ExtraErrorData(),
+      new Sentry.BrowserTracing(),
     ],
     release,
+    /**
+     * Setting a value for `tracesSampleRate` enables performance monitoring in Sentry.
+     * Once performance monitoring is enabled, transactions are sent to Sentry every time
+     * a user loads a page or navigates within the app.
+     * Since the amount of traffic the app gets is important, this means a lot of
+     * transactions are sent. By setting `tracesSampleRate` to a value lower than 1.0, we
+     * reduce the volume of transactions to a more reasonable amount.
+     */
+    tracesSampleRate: 0.1,
```

Comment:
Mobile is using a sampling rate of 0.05, and even that small a rate is a huge amount of events. We should consider adjusting this based upon our current account limits, also taking into consideration the userbase between extension and mobile.

Reply:
Given `tracesSampleRate` is currently set to 0.04 for Mobile and given that the amount of MAU is more than 3x more important for Extension, I guess we can set `tracesSampleRate` to 0.01 for Extension.